dev-qt/qtdeclarative/qtdeclarative-5.15.3.9999.ebuild

```diff
@@ -56,3 +56,8 @@ src_configure() {
 	)
 	qt5-build_src_configure
 }
+
+src_install() {
+	qt5-build_src_install
+	qt5_symlink_binary_to_path qml 5
```

Comment:
I believe there's a few more here... in Qt6 the following are considered user-facing:

```
$ git grep qt_internal_add_app
tools/qml/CMakeLists.txt:qt_internal_add_app(qml
tools/qmleasing/CMakeLists.txt:qt_internal_add_app(qmleasing
tools/qmljs/CMakeLists.txt:qt_internal_add_app(qmljs
tools/qmlpreview/CMakeLists.txt:qt_internal_add_app(qmlpreview
tools/qmlscene/CMakeLists.txt:qt_internal_add_app(qmlscene
```

(`qmlscene` is deprecated in favor of `qml` in Qt6, but that's none of our business)

I don't know if the same tools should also be considered user-facing (and thus symlinked with suffix) in Qt5, but it seems a good starting point?

(and no, the classification criteria still don't make any sense to me... why is qmljs user-facing? sure, it can be manually invoked by a developer using Qt, but the same applies to qmllint, qmlprofiler, and several others...)

Reply:
I need to work on the draft as well to not confuse what we mean with user-facing... Anyway, yes let's symlink those others too then.

Reply:
well, upstream uses USER_FACING in their cmake files... but yes, I agree on aligning with whatever upstream does.